scripts/ci/crates.py

```diff
@@ -29,21 +29,21 @@
 from enum import Enum
 from glob import glob
 from pathlib import Path
-from time import sleep, time
 from typing import Any, Generator
 
 import git
 import requests
 import tomlkit
 from colorama import Fore
 from colorama import init as colorama_init
+from dag import DAG
 from semver import VersionInfo
 
 CARGO_PATH = shutil.which("cargo") or "cargo"
 DEFAULT_PRE_ID = "alpha"
 
 
-def cargo(dry_run: bool, args: str, cwd: str | Path | None = None, env: dict[str, Any] = {}) -> Any:
+def cargo(args: str, cwd: str | Path | None = None, env: dict[str, Any] = {}, dry_run: bool = False) -> Any:
```

Comment:
tip: force callers to actually name the arguments to the function with this one weird trick

```suggestion
def cargo(args: str, *, cwd: str | Path | None = None, env: dict[str, Any] = {}, dry_run: bool = False) -> Any:
```